test/e2e-tests.sh

```diff
@@ -19,11 +19,12 @@
 
 source $(git rev-parse --show-toplevel)/test/e2e-common.sh
 
-
 # Setting defaults
 PIPELINE_FEATURE_GATE=${PIPELINE_FEATURE_GATE:-stable}
 SKIP_INITIALIZE=${SKIP_INITIALIZE:="false"}
 RUN_YAML_TESTS=${RUN_YAML_TESTS:="true"}
+E2E_CLUSTER_MACHINE=${E2E_CLUSTER_MACHINE:-n1-standard-8}
```

Comment:
@dibyom noticed that the builds for this PR are still creating clusters with the default values - I am pretty sure this is because we source `test/e2e-common.sh` before setting `E2E_CLUSTER_MACHINE` and `E2E_MAX_CLUSTER_NODES`. Since `test/e2e-common.sh` itself sources Plumbing's `e2e-tests.sh`, which sets those vars to the defaults as readonly, trying to set them later doesn't actually do anything. So in general, feels like the `Setting defaults` stanza should be before the `source`.